test/models/return-version-model.test.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseSupport = require('../support/database.js')
+const ReturnVersionHelper = require('../support/helpers/return-version.helper.js')
+
+// Thing under test
+const ReturnVersionModel = require('../../app/models/return-version.model.js')
+
+describe('Return version model', () => {
+  let testRecord
+
+  beforeEach(async () => {
+    await DatabaseSupport.clean()
+  })
+
+  describe('Basic query', () => {
```

Comment:
Do we need to add some tests for the models relationships to `licence.model` and `return-requirement-point.model`